include/zephyr/fs/fs.h

```diff
@@ -452,8 +452,9 @@ int fs_sync(struct fs_file_t *zfp);
  * @param path Path to the directory to create
  *
  * @retval 0 on success;
- * @retval -EROFS if file is read-only, or when file system has been mounted
- *	   with the FS_MOUNT_FLAG_READ_ONLY flag;
+ * @retval -EEXIST if entry of given name exists;
+ * @retval -EROFS if path is within is read-only directory, or when
+ *         file system has been mounted with the FS_MOUNT_FLAG_READ_ONLY flag;
```

Comment:
I guess that changes about `-EROFS` were not intended.

Reply:
No, it was intentional because there was copy-paste error from the last time when doc has been reworked and I, probably, have used "file" to describe what supposed to be path.

Reply:
But still "if path is within is read-only directory" does not sound right. Unwanted "is" word?

And if this is intentional, then would be good to write it down in commit message (or even better, separate commit) :)

Reply:
> But still "if path is within is read-only directory" does not sound right. Unwanted "is" word?
> 
> And if this is intentional, then would be good to write it down in commit message (or even better, separate commit) :)

Will fix. Will fix.